src/main/java/school/hei/haapi/endpoint/rest/security/SecurityConf.java

```diff
@@ -143,6 +143,8 @@ req, res, null, forbiddenWithRemoteInfo(req))))
                     .hasAnyRole(MANAGER.getRole(), TEACHER.getRole(), STUDENT.getRole())
                     .requestMatchers(GET, "/school/files/*")
                     .hasAnyRole(MANAGER.getRole(), STUDENT.getRole(), TEACHER.getRole())
+                    .requestMatchers(POST, "/school/files/share_link")
+                    .hasAnyRole(MANAGER.getRole(), STUDENT.getRole(), TEACHER.getRole())
```

Comment:
info: what are we supposed to do with "non school-files" ? 
( this is a non-blocking comment, I would simply like to get more informations)